imsim/readout.py

```diff
@@ -268,16 +267,21 @@ def get_primary_hdu(eimage, lsst_num, camera_name=None,
         phdu.header['SENSNAME'] = sensor
         phdu.header['RATEL'] = ratel
         phdu.header['DECTEL'] = dectel
-    elif camera_name == 'LsstComCam':
+        telcode = 'MC'
+    elif camera_name in ('LsstComCam', 'LsstComCamSim') :
```

Comment:
I'm wondering if we should omit `LsstComCam` as an option since the DM folks would like to distinguish between simulated and real data by the instrument model.  @jmeyers314 Any thoughts?

Reply:
That seems like a reasonable default position to me.  We can revisit if it breaks any workflows later.  Presumably the same applies to `LsstCam` / `LsstCamSim` ?

Reply:
Thanks, I'll remove the `LsstComCam` option then, and once I find time to make a `LsstCamSim` instrument model, I'll replace `LsstCam` similarly. BTW, feel free to comment and/or review this PR. I just picked on Chris because he's familiar with the header and tree ring discussions.